java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -1820,6 +1824,25 @@ public Table filter(ColumnView mask) {
     return new Table(filter(nativeHandle, mask.getNativeView()));
   }
 
+  /**
+   * Copy rows of the current table to an output table if the corresponding rows in the keys columns
+   * are unique. The keys columns are a subset of the current table columns and their indices are
+   * specified by an input array.
+   *
+   * @param keyColumns Array of indices representing key columns from the current table.
+   * @param keepFirst If it is true, the first element in a sequence of duplicate elements will be
+   *                  copied. Otherwise, copy the last element.
+   * @param nullsEqual Flag to denote whether nulls are treated as equal.
+   * @param nullsBefore Flag to specify whether nulls should appear before or after non-null elements.
```

Comment:
This implies there's an ordering that can be relied upon in the resulting table.  Can we elaborate in the description above what the caller can assume about the order of keys in the resulting table?  As the docs are written now, I could see a caller assuming the relative positions of preserved keys are unchanged.

Reply:
Thanks. I have updated the comments, clarifying things.